src/components/calorie-tracker/RapidMealLogger.tsx

```
@@ -0,0 +1,406 @@
+            break;
+          case 'f':
+            e.preventDefault();
+            openFavorites();
+            break;
+        }
+      }
+    };
+
+    window.addEventListener('keydown', handleKeyPress);
+    return () => window.removeEventListener('keydown', handleKeyPress);
+  }, [handleQuickPhoto, handleVoiceEntry, openFavorites]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)